### PR TITLE
Front

### LINKED ISSUE #51: Complete wordcloud demo
DOD:
- [x] Wordcloud demo created and ready for presentation on ZPP

### PR DESCRIPTION
**Pull request by kowaalczyk**
Closes: https://github.com/spacy-pl/utils/issues/51

@Gizzio take a look at readme before running it, there is one new thing necessary to do before build.
Index template and lib.js are taken from previous demo with minor changes, this code is not the prettiest but I would suggest to move forward with demo implementation and leave the cleaning process for later, when we have a presentable demo :)

### CODE REVIEW

File: web/download_static.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+def download_zip(url, dest):
+    r = requests.get(url)
+    if not os.path.exists(dest):
+        os.makedirs(dest)
+    with zipfile.ZipFile(io.BytesIO(r.content)) as thezip:
+        thezip.extractall(dest)
+
+
+if __name__ == '__main__':
+    print(static_dir)
```
Comment (Gizzio) [inline web/download_static.py:50]: I suggest to log something more here about it(for example what this path is) or to remove this line.

File: web/static/src/demo.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            elementId: `${demoEventName}`,
+            eventName: 'submit'
+        }],
+        handler,
+        `${demoEventName}-input`,
+        `${demoEventName}-output`
+    );
+}
+
+// UIkit.util.ready(() => {
```
Comment (Gizzio) [inline web/static/src/demo.js:132]: Do we need this code?

File: web/templates/index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          content="website" />
+    <meta property="og:title"
+          content="spaCy.pl" />
+    <meta property="og:description"
+          content="Developing tools for complete Polish language support in spaCy." />
+    <meta property="og:image"
+          content="{{ url_for('static', filename='png/hero.jpg') }}" />
+
+    <title>
+        spaCy.pl
```
Comment (Gizzio) [inline web/templates/index.html:21]: change to spaCy-pl (to not confuse people)

File: web/static/src/demo.js (hunk 2 above)
Comment (kowaalczyk) [reply web/static/src/demo.js:132]: It will be useful when merging with the other demo.

File: web/templates/index.html (hunk 3 above)
Comment (kowaalczyk) [reply web/templates/index.html:21]: We'll leave that to merging with the other demo (the website) as this is not even visible in this version.

**Pull request merged**